addons/l10n_in_edi/views/res_config_settings_views.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="res_config_settings_view_form_inherit_l10n_in_edi" model="ir.ui.view">
+        <field name="name">res.config.settings.form.inherit.l10n_in_edi</field>
+        <field name="model">res.config.settings</field>
+        <field name="inherit_id" ref="iap.res_config_settings_view_form"/>
+        <field name="arch" type="xml">
+            <div id="iap_portal" position="inside">
+                <div class='row mt16 o_settings_container' name="l10n_in_edi_iap" attrs="{'invisible': [('country_code', '!=', 'IN')]}">
+                    <div class="col-12 col-lg-6 o_setting_box" id="gsp_setting">
+                        <div class="o_setting_right_pane">
+                            <t class="o_form_label">Indian E-invoice IAP</t>
+                            <span class="fa fa-lg fa-building-o" title="Values set here are company-specific." aria-label="Values set here are company-specific." groups="base.group_multi_company" role="img"/>
+                            <div class="text-muted">
+                                Setup E-invoice IAP Service for this company
+                            </div>
+                            <div class="text-warning" attrs="{'invisible': [
+                                '|',('l10n_in_edi_username','in', (False,'')),
+                                    ('l10n_in_edi_token','not in', (False,'')),
+                                ]}">
+                                Click on confirm to varify username and password
+                            </div>
+                            <div class="content-group">
+                                <div class="mt16 row">
+                                    <label for="l10n_in_edi_username" string="Username" class="col-3 col-lg-3 o_light_label"/>
+                                    <field name="l10n_in_edi_username"  nolabel="1"/>
+                                    <label for="l10n_in_edi_password" string="Password" class="col-3 col-lg-3 o_light_label" />
+                                    <field name="l10n_in_edi_password" password="True" nolabel="1"/>
+                                    <field name="l10n_in_edi_token" invisible="1"/>
```

Comment:
Why do we need token in the view?

Reply:
we show and hide warning text like "Click on Test to varify username and password"
if token is there it mins that username and password is varifyed

Reply:
as per your next point I remove it.